### PR TITLE
fix: docs update, doc additional env vars

### PR DESCRIPTION
**Pull request by jeefy**
Signed-off-by: Jeffrey Sica <me@jeefy.dev>

cc @marshallofsound

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +104,9 @@ The following environment variables represent the configuration of the actual Sh
 | `GSUITE_TOKEN` | | GSuite authentication token | `gsuite` |
 | `SHERIFF_GSUITE_DOMAIN` | | The primary domain of your GSuite account | `gsuite` `slack` |
 | `SHERIFF_SLACK_DOMAIN` | | The "domain" part of `{domain}.slack.com` for your Slack instance  | `gsuite` if you add slack email addresses to your google groups for notifications |
+| `ORGANIZATION_NAME` | | GitHub org that the GitHub App has been installed to | `electron` |
+| `REPO_NAME` | | GitHub repo that the GitHub App has been installed to  | `electron` |
```
Comment (MarshallOfSound) [inline README.md:108]: Wondering if these variables should ever be different from `PERMISSIONS_FILE_ORG` and `PERMISSIONS_FILE_REPO` 🤔 

i.e. I don't think there's a usecase for the github app to be installed in a different org than the one containing the `permissions` repo.

Would your setup have these vars as the same thing? If so we should probably just dedupe them 😅

**Comment by jeefy** [comment]
The org is the same but the app isn't installed in a specific repo.

Additionally I do need to set the repo to be different than `.permissions`
in our use case.

I'm +1 for deduping though, it's cleaner.

**Comment by MarshallOfSound** [comment]
> but the app isn't installed in a specific repo.

Yeah this is a side effect of the way I wrote `@electron/github-app-auth`.  Just added new `forOrg` helpers so you don't need to pass in a specific repo to get the installation information.  I'll push it up to your PR

**Review by MarshallOfSound — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -45,6 +45,18 @@ You want to specify the following options:
 
 Finally, click "Add webhook".
 
+### The GitHub App
+
+To manage GitHub instances, Sheriff requires you to create a GitHub App that gets installed in the desired Org.
```
Comment (MarshallOfSound) [inline README.md:50]: We should also probably document what permissions the app needs.  Specifically:

```md
Org:
administration:write
contents:read
metadata:read

Repo:
members:write
```

Comment (jeefy) [reply README.md:50]: ezpz

**Comment by jeefy** [comment]
Added the permissions/scopes you provided into the docs, should be g2g!

**Review by MarshallOfSound — APPROVED** [review]

**Pull request merged**